config/samples/translator-binding.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: ibmcloud.ibm.com/v1alpha1
 kind: Binding
 metadata:
   name: binding-translator
+  namespace: name2
 spec:
   serviceName: mytranslator
+  serviceNamespace: name1
```

Comment:
Can we have perhaps a separate sample to illustrate this feature ? (and there point to `default`, so if you create the binding in a different ns it will use the service created in default)

Reply:
Not sure how to do this without having a non-existent namespace... (?)

Reply:
for example, create translator-binding-with-ns.yaml which can be created in any namespace other than default with `kubectl -n <ns> translator-binding-with-ns.yaml ` and have that point to `serviceNamespace: default`

Reply:
created translator-binding-with-ns.yaml which should be created in namespace ns and points to a service in default